deploy/helm/values.yaml

```diff
@@ -9,6 +9,7 @@ replicaCount: 1
 image:
   repository: quay.io/kubernetes_incubator/nfs-provisioner
   tag: v2.3.0
+  digest: ""
```

Comment:
Thanks for your PR!
Could you please bump chart version and add `digest` into the [configuration reference](https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/tree/master/deploy/helm#configuration)

Reply:
Done, I bumped the minor version as it was a new functionality for the chart even though it was very minor.